package.json

```diff
@@ -164,8 +164,8 @@
 		"@storybook/addon-viewport": "^6.3.0",
 		"@storybook/react": "^6.3.0",
 		"@svgr/webpack": "^5.4.0",
-		"@testing-library/jest-dom": "^5.11.0",
-		"@testing-library/react": "^10.4.5",
+		"@testing-library/jest-dom": "^5.12.0",
```

Comment:
Other packages that also needed upgrading. It was months ago since I did this ticket though so I can't remember exactly why :joy: